tests/integ/scala/test_dataframe_suite.py

```diff
@@ -865,9 +865,10 @@ def test_df_stat_sampleBy_seed(session, caplog, use_simplified_query_gen):
 
     # DataFrame doesn't work with seed
     caplog.clear()
-    with caplog.at_level(logging.WARNING):
-        sample_by_action(TestData.monthly_sales(session))
-    assert "`seed` argument is ignored on `DataFrame` object" in caplog.text
```

Comment:
@sfc-gh-jdu I saw your wrote the tests. in our sproc regression test, the assertion doesn't stand anymore, is this expected?

> assert "`seed` argument is ignored on `DataFrame` object" in caplog.text\nE       AssertionError: assert \'`seed` argument is ignored on `DataFrame` object\' in \'\'\nE        +  where \'\' = &lt;_pytest.logging.LogCaptureFixture object at 0x7ee5f19bb5e0&gt;